tox.ini

```diff
@@ -74,17 +70,55 @@ usedevelop = false
 commands =
     python -m yapf -i -r molecule/ test/
 deps = yapf>=0.25.0,<0.27  # pyup: < 0.27 # disable updates, conflicts with flake8 as per https://github.com/ansible/molecule/pull/1915
-extras =
-skip_install = true
-usedevelop = false
+extras = {[testenv:lint-base]extras}
+skip_install = {[testenv:lint-base]skip_install}
+usedevelop = {[testenv:lint-base]usedevelop}
 
 [testenv:format-check]
+description = Run yapf check
 commands =
     python -m yapf -d -r molecule/ test/
 deps = {[testenv:format]deps}
-extras = {[testenv:format]extras}
-skip_install = true
-usedevelop = false
+extras = {[testenv:lint-base]extras}
+skip_install = {[testenv:lint-base]skip_install}
+usedevelop = {[testenv:lint-base]usedevelop}
+
+[testenv:flake8]
```

Comment:
https://github.com/ansible/molecule/pull/1996 is the reson why I do not fancy this direction. Having one extra tox environment for each linter does not scale well. That change allows us to have only one tox environment for linting while being able to isolate each linter and still be able to run only one of them if we want.

Reply:
This is unrelated to current PR.